PocketKit/Sources/PocketKit/MyList/SavesContainerViewController.swift

```diff
@@ -87,7 +88,6 @@ class SavesContainerViewController: UIViewController, UISearchBarDelegate {
 
         view.accessibilityIdentifier = "saves"
         select(child: viewControllers.first)
-        navigationController?.delegate = self
```

Comment:
nav controller was nil here.

Reply:
This is important, because our Navcontroller delegate contains the logic to unselect an item in the list... not changing that now because I'd rather see this move to SwiftUI